.github/workflows/release-please.yml

```diff
@@ -0,0 +1,23 @@
+on:
```

Comment:
Nit, can we make spaces formatting consistent?

Reply:
Yes, good spot! Fixed